snapcraft.yaml

```diff
@@ -0,0 +1,30 @@
+name: dvc
+summary: Data Version Control
+description: Git for Data & Models https://dvc.org
+adopt-info: dvc
+grade: stable
+confinement: classic  # for same reasons as snap `git-ubuntu`
+base: core18
+license: Apache-2.0
+parts:
+  dvc:
+    plugin: python
+    python-packages: [wheel]
+    source: .
+    build-packages:
+    - git
+    - libc-dev
+    stage-packages:
+    - git
+    - libc-dev
```

Comment:
presumably only runtime ranter than `-dev` required here

Reply:
also @efiop any idea about other runtime dependencies? `ssh` libraries etc?

Reply:
@casperdcl I'm not quite sure about what base stuff snap images have, but I don't think we need ssh libraries, as paramiko is pure-python. Probably need openssl though.

Reply:
@casperdcl Btw, will this currently install `dvc[all]` or `dvc`(no options)?

Reply:
just `dvc`. It's in the checklist to do the rest...

Reply:
done methinks